### ISSUE TITLE
Arrow key content hierarchy navigation doesn't work when there are multiple projects

### ISSUE THREAD

**Issue by thomasheartman** [enonic/app-contentstudio#3222]
## Issue

Using arrow keys to navigate the content hierarchy doesn't work if the user must first select a project.

## Steps to reproduce

1. Open an XP instance with only a single CS project. The project should have some content to test navigation.
2. Navigate to CS.
3. Notice that the arrow keys allow you ta traverse the tree and expand nodes
4. Create a new CS project
5. Return to CS (/reload)
6. Select the original project (with content) when prompted
7. Try to navigate the hierarchy with arrow keys and notice that it doesn't work.

## Context

I've been having an issue for a while where using the arrow keys to navigate content just doesn't work. When this happens I can't seem to make it work again. While this _definitely also happens in other cases_, I was able to isolate _one_ case where this seems to happen consistently.

## Update / further specification

I think I've managed to narrow it down further. If the user reloads the page, it seems that the arrow keys start working, even if they switch projects. Right now I can only reproduce it (reliably?) when navigating to CS from the XP landing page via the admin menu.

**Comment by reisfmb** [comment]
Fixed in another issue.
